views/premium.html

```diff
@@ -5,10 +5,15 @@
 
   <% if display_discount %>
   <div class="mailpoet-discount-container">
-    <h1><%= __('Discounts up to 40% for a limited time') %></h1>
+    <h1><%= __('Discounts up to 50% for a limited time') %></h1>
     <p><%= __('Don’t miss out on our only sale of the year which ends on November 30.') %></p>
     <p><%= __('There’s never been a better time to change gears.') %></p>
-    <a href=" https://account.mailpoet.com/?s=<%= subscriber_count %>&utm_source=plugin&utm_medium=premium&utm_campaign=discount">Save up to 40%</a>
+    <a
+      href="https://account.mailpoet.com/?s=<%= subscriber_count %>&utm_source=plugin&utm_medium=premium&utm_campaign=discount"
+      class="button button-primary"
```

Comment:
I am not sure this is the design we want. What do you think @kgjerstad ?

![dd](https://user-images.githubusercontent.com/29855930/47154806-9be9cc80-d2e3-11e8-8fa7-fa28e76cdd28.png)

Reply:
Well, this is what Quentin wrote to specification...

Reply:
I suggest we copy the style of the "Purchase Now" button at the bottom of the same page: https://d.pr/i/HHlq36